not able to build opm-common on mac

Hei ... try to build opm-common but got some issues .. any one can help. ?
```
[ 15%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Python/Python.cpp.o
In file included from /Users/miroine/project_developments/repos/opm-common/src/opm/parser/eclipse/Python/Python.cpp:21:
/Users/miroine/project_developments/repos/opm-common/opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp:298:33: error: 
      no matching member function for call to 'splitDynMap'
            auto splitvfpprod = splitDynMap(vfpprod_tables);
                                ^~~~~~~~~~~
/Users/miroine/project_developments/repos/opm-common/opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp:452:9: note: 
      candidate template ignored: substitution failure : template template
      argument has different template parameters than its corresponding template
      template parameter
        splitDynMap(const Map<Key, Opm::DynamicState<Type>>& map)
        ^
/Users/miroine/project_developments/repos/opm-common/opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp:301:32: error: 
      no matching member function for call to 'splitDynMap'
            auto splitvfpinj = splitDynMap(vfpinj_tables);
                               ^~~~~~~~~~~
/Users/miroine/project_developments/repos/opm-common/opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp:452:9: note: 
      candidate template ignored: substitution failure : template template
      argument has different template parameters than its corresponding template
      template parameter
        splitDynMap(const Map<Key, Opm::DynamicState<Type>>& map)
        ^
2 errors generated.
make[2]: *** [CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/Python/Python.cpp.o] Error 1
make[1]: *** [CMakeFiles/opmcommon.dir/all] Error 2
make: *** [all] Error 2
```

--- a/opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp
+++ b/opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp
@@ -295,10 +295,10 @@ namespace Opm
             m_tuning.serializeOp(serializer);
             m_messageLimits.serializeOp(serializer);
             m_runspec.serializeOp(serializer);
-            auto splitvfpprod = splitDynMap(vfpprod_tables);
+            auto splitvfpprod = splitDynMap<Map2>(vfpprod_tables);
             serializer.vector(splitvfpprod.first);
             serializer(splitvfpprod.second);
-            auto splitvfpinj = splitDynMap(vfpinj_tables);
+            auto splitvfpinj = splitDynMap<Map2>(vfpinj_tables);
             serializer.vector(splitvfpinj.first);
             serializer(splitvfpinj.second);
             wtest_config.serializeOp(serializer);
@@ -317,12 +317,13 @@ namespace Opm
             if (!serializer.isSerializing()) {
                 reconstructDynMap(splitWells.first, splitWells.second, wells_static);
                 reconstructDynMap(splitGroups.first, splitGroups.second, groups);
-                reconstructDynMap(splitvfpprod.first, splitvfpprod.second, vfpprod_tables);
-                reconstructDynMap(splitvfpinj.first, splitvfpinj.second, vfpinj_tables);
+                reconstructDynMap<Map2>(splitvfpprod.first, splitvfpprod.second, vfpprod_tables);
+                reconstructDynMap<Map2>(splitvfpinj.first, splitvfpinj.second, vfpinj_tables);
             }
         }
 
     private:
+        template<class Key, class Value> using Map2 = std::map<Key,Value>;
         TimeMap m_timeMap;
         WellMap wells_static;
         GroupMap groups;

